tests/mgbench/runners.py

```diff
@@ -418,6 +524,9 @@ def stop_db(self, arg):
     def clean_db(self):
         pass
 
+    def get_database_port(self):
+        return self._bolt_port
```

Comment:
This method is part of abstract class but _bolt_port isn't declared in abstract class

Reply:
still not resolved